homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
```

Comment:
Can we be more specific in `get_device_info`?

Reply:
More specific with regards to... catching more specific exceptions than Exception, or do you have something else in mind?

Reply:
Oh sorry, yes can we catch more specific exceptions?